<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CENTAUR_NAMESPACE::SplashDialog</class>
 <widget class="QDialog" name="CENTAUR_NAMESPACE::SplashDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CENTAUR_NAMESPACE::MovableFrame" name="logMainFrame">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>340</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>550</width>
       <height>340</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#logMainFrame{
	border-image: url(:/splash/splash)  0 0 0 0 stretch stretch;

border-top-left-radius: 10px;
border-top-right-radius: 10px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
border-left: 1px solid rgb(92,92,92);
border-top: 1px solid rgb(104,104,104);
border-right: 1px solid rgb(104,104,104);
border-bottom: 1px solid rgb(92,92,92);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="3" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar {
	border:  0px;
    border-radius: 5px;
	background-color: rgba(3,12,19,125);
}
QProgressBar::chunk {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(72	,194,125, 255), stop:1 rgba(20,108,255,255));
	border-radius: 5px;
}
</string>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:  rgb(3,12,19);
	font-weight: 800;
	font-size: 14px;
}</string>
        </property>
        <property name="text">
         <string>Loading...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="displayText">
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:  rgb(38,46,52);
	font-weight: 600;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="indent">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CENTAUR_NAMESPACE::MovableFrame</class>
   <extends>QFrame</extends>
   <header>../../Library/cui/include/MovableFrame.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
